src/server/wesnothd/server.hpp

```diff
@@ -105,6 +106,7 @@ class server : public server_base
 
 private:
 	wesnothd::ban_manager ban_manager_;
+	randomness::mt_rng rng_;
```

Comment:
We already have a MT RNG member in `die_`. I think we can just reuse that? We could change the name since it's not just used for dice rolling then.

Reply:
Is there a reason it uses `std::mt19937` directly rather than the `randomness::mt_rng` wrapper?